assets/stylesheets/bootstrap/_type.scss

```diff
@@ -136,7 +137,7 @@ mark,
 // -------------------------
 
 .page-header {
-  padding-bottom: (($line-height-computed / 2) - 1);
+  padding-bottom: math.div(($line-height-computed, 2) - 1);
```

Comment:
should be `padding-bottom: (math.div($line-height-computed, 2) - 1);`

Reply:
oh, christmas. I suppose I will need to write a proper parser after all.

Reply:
@AprilArcus Any plans to make the two fixes outlined by @aEvgenn ?

Reply:
Look at `kikyous/bootstrap3-sass` . Suitable for a temporary solution. There will be only one trouble - replacing the package name in the paths `bootstrap-sass` -> `bootstrap3-sass`

Reply:
> @AprilArcus Any plans to make the two fixes outlined by @aEvgenn ?

yes

Reply:
fixed